taskfile/read/taskfile.go

```diff
@@ -22,11 +22,20 @@ var (
 )
 
 // Taskfile reads a Taskfile for a given directory
+// Uses Taskfile.yaml or Taskfile.yml when entrypoint is left empty
 func Taskfile(dir string, entrypoint string) (*taskfile.Taskfile, error) {
-	path := filepath.Join(dir, entrypoint)
-	if _, err := os.Stat(path); err != nil {
-		return nil, fmt.Errorf(`task: No Taskfile found on "%s". Use "task --init" to create a new one`, path)
+	if dir == "" {
+		d, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
+		dir = d
+	}
```

Comment:
I think this whole `if` block can just go. `filepath.Join(wd, "foo")` and `filepath.Join("", "foo")` should be equivalent. What do you think? Any particular reason on while you added it?

Reply:
`exits(path string)` needs a path and won't work with an empty string. `dir` and `entrypoint` are both "" when user doesn't specify anything as the default filename handling was removed from other places and is defined only in the read package. 

That didn't come up in tests as they all specify a dir. But the problem is seen here: https://github.com/go-task/task/runs/4665736655?check_suite_focus=true